src/vehicle.h

```diff
@@ -1238,6 +1238,7 @@ class vehicle
         bool is_engine_type_on( int e, const itype_id &ft ) const;
         //returns whether the engine is enabled or not
         bool is_engine_on( int e ) const;
+        bool is_engine_on() const;
```

Comment:
this works out to vehicle->engine_on, and engine_on is public, so is this really necessary?

Reply:
Nope totally not.  Somehow I thought it was private.  Will fix.